setup.bash

```diff
@@ -66,7 +82,3 @@ if [ -f $TUE_DIR/setup/tue-misc.bash ]
 then
     source $TUE_DIR/setup/tue-misc.bash
```

Comment:
May be merge this with tue-functions.bash as both just have one function each in them.